students/mariusz_michalczyk/lesson_08_regex/phone_nr_validation.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def validate_phone_nr(nr):
+    regex = re.compile(r'(\+\d{2})*\s*\-*(\d{3})\s*\-*(\d{3})\s*\-*(\d{3})')
```

Comment:
`+48 +48  +11 +11   ---   111   ---111      ------111` will match. For 0 or 1 repetitions use `?`, for the dashes you can either do `[ -]` or `( |-)`.